functions/src/import/index.ts

```diff
@@ -94,14 +98,15 @@ async function retrieveCollectionFromFirestore(
  * Adapts documents and indexes them in Meilisearch.
  * @param {any} docs
```

Comment:
Why we use `any` instead of `DocumentSnapshot[]` here?

Reply:
I thought the previous one was a mistake `@param {Change<DocumentSnapshot>} change` since `change` is not `docs`. But you'r right, putting it back :)